File: solutions/zig/two-fer/1/two_fer.zig
const std = @import("std");
const fmt = std.fmt;

pub fn twoFer(buffer: []u8, name: ?[]const u8) anyerror![]u8 {
    const n = name orelse "you";
    return fmt.bufPrint(buffer, "One for {s}, one for me.", .{n});
}

File: solutions/go/two-fer/2/two_fer.go
// Package twofer is short for two for one. One for you and one for me.
package twofer

// ShareWith return one for `name` and one for me.
func ShareWith(name string) string {
	if name == "" {
		name = "you"
	}
	return "One for " + name + ", one for me."
}
